Automattic/jetpack
# AI Assistant: Add language reminder to action prompts

Fixes #31820

## Proposed changes:

* Adds a rule to the text of the toolbar options prompts.

After many tests, adding it to the rules array had little to no effect, but adding it directly to the message seems to work very well. Maybe we'll need to rethink where rules are defined.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add an AI Assistant block
* Ask for some content in a language other than English
* Iterate over the content in its language with the toolbar options
* Check that the content is not translated to English at any point
* Translate the content to English
* Check that the next iterations return English as normal
* Check for any regressions
